tests/testthat/_snaps/bump_version/NEWS.md

```diff
@@ -0,0 +1,10 @@
+<!-- NEWS.md is maintained by https://cynkra.github.io/fledge, do not edit -->
```

Comment:
I added a snapshot test _with_ but not _of_ the new helper function, is this what you meant? I could not think of a relevant snapshot test _of_ the new helper function.